wxt.config.ts

```diff
@@ -54,6 +54,7 @@ const getManifest: ((env: ConfigEnv) => Partial<Manifest.WebExtensionManifest>)
       resources: ["inject.js", "ch_patterns.json"],
       matches: ["*://*.buff.163.com/*"]
     }],
+    omnibox: { keyword : "buff" },
```

Comment:
Using "bb" as the keyword for "BetterBuff" might be better.

Reply:
On the other hand, having "buff" as keyword should lead to users discovering this feature more naturally and on their own.